### PR TITLE
feat(alert): add attributes min & max to alert inputs

### PR DESCRIPTION
**Pull request by TomDemulierChevret**
#### Short description of what this resolves:
We cannot set currently min & max attributes for inputs used in Alerts.
It can be quite useful for input of type "date".

#### Changes proposed in this pull request:
- add min & max attributes to AlertCmp
- add an input date with min & max set to the alert e2e test (function doPrompt())
- add an input date without min nor max set to the alert e2e test (function doPrompt())

**Ionic Version**: 2.1.0

**Fixes**: #10590

### CODE REVIEW

**Comment by manucorporat** [comment]
@TomDemulierChevret  have you ensured this works?

**Comment by TomDemulierChevret** [comment]
@manucorporat It does work in browser (the e2e test includes a text "date" input with min & max set, a text "date" input without min & max set) but I didn't know how to test it easily on mobile.

**Comment by manucorporat** [comment]
That would be super useful, so we can merge this as soon as possible.
Now we are happy to start merging new features since the framework is stabilizing and the final version was released.

Let's me know what you find :)

File: src/components/alert/alert.ts [hunk 1]
```diff
@@ -244,6 +244,8 @@ export class Alert extends ViewController {
  *  | label       | `string`  | The input's label (only for radio/checkbox inputs)              |
  *  | checked     | `boolean` | Whether or not the input is checked.                            |
  *  | id          | `string`  | The input's id.                                                 |
+ *  | min         | `string`  | The input's minimum authorized value (only for date inputs)     |
```
Comment (manucorporat) [inline src/components/alert/alert.ts]: It might work with "number" inputs too. can you try?

Comment (TomDemulierChevret) [inline src/components/alert/alert.ts]: It does work with number inputs too.

**Comment by TomDemulierChevret** [comment]
I just tested on Android 7.1.1 (Nexus 5X) it does work.
But it doesn't seem to work on iOS (10.2.1, iPhone 6) : the native datepicker works but the values are not bound within the min & max set.

**Comment by syedatifgd** [comment]
Kuttay k buchon, it does not work. Fuckers.

**Pull request merged**